### DISCUSSION TITLE
Assigning radiation model using pyAEDT in Icepak

### DISCUSSION THREAD

**Discussion by Somayaji1995** [ansys/pyaedt#4862] [Q&A] [answered]
I want to assign Discrete Ordinate Radiation model in my simulation using script, Is there any specific way where in I can model this ?

Methods I have tried is listed below, none of it is working or giving any error message
my_setup.props["Radiation Model"]= "On"
&
my_setup.props["Radiation Model"]= "DiscreteOrdinates"

**Comment by ansys-satyajeet** [comment, marked as answer]
Hi, 

You can assign Discrete Ordinate Radiation model using the lines below

```python
setup = ipk.create_setup("sim1")
setup.props["Radiation Model"] = "Discrete Ordinates Model"
setup.props["Flow Iteration Per Radiation Iteration"] = "10"
setup.props["ThetaDivision"] = 2
setup.props["PhiDivision"] = 2
setup.props["ThetaPixels"] = 2
setup.props["PhiPixels"] = 2
setup.update()
```

In the code snippet above, you will also find the keys for other settings such as resolution for DO model and flow iteration per radiation iteration.
